lib/commands/log.js

```diff
@@ -47,7 +47,7 @@ extensions.startLogCapture = async function startLogCapture () {
       this.logs.syslog = new IOSSimulatorLog({
         sim: this.opts.device,
         showLogs: this.opts.showIOSLog,
-        xcodeVersion: this.xcodeVersion,
```

Comment:
is xcodeVersion not used?

Reply:
It's not used in this repo. Could it be used elsewhere? Maybe I'll add it back to be safe